v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/CsvConverters.java

```diff
@@ -0,0 +1,540 @@
+                  return CharStreams.toString(reader);
+                } catch (IOException e) {
+                  throw new UncheckedIOException(e);
+                }
+              })
+          .collect(Collectors.joining());
+    } catch (IOException ioe) {
+      throw new RuntimeException("File system i/o error: " + ioe.getMessage());
+    }
+  }
+
+  /**
+   * Builds Json string from list of values and headers or values and schema if schema is provided.
+   *
+   * @param headers optional list of strings which is the header of the Csv file.
+   * @param values list of strings which are combined with header or json schema to create Json
+   *     string.
+   * @param jsonSchemaString
+   * @return Json string containing object.
+   * @throws IOException thrown if Json object is not able to be written.
+   */
+  static String buildJsonString(
+      @Nullable List<String> headers, List<String> values, @Nullable String jsonSchemaString)
+      throws IOException {
+    StringWriter stringWriter = new StringWriter();
+    JsonWriter writer = new JsonWriter(stringWriter);
+    if (jsonSchemaString != null) {
+      JsonArray jsonSchema = new JsonParser().parse(jsonSchemaString).getAsJsonArray();
+      writer.beginObject();
+      for (int i = 0; i < jsonSchema.size(); i++) {
+        JsonObject jsonObject = jsonSchema.get(i).getAsJsonObject();
+        writer.name(jsonObject.get("name").getAsString());
+        switch (jsonObject.get("type").getAsString().toUpperCase()) {
+          case "LONG":
+            writer.value(Long.parseLong(values.get(i).toString()));
+            break;
+
+          case "DOUBLE":
+            writer.value(Double.parseDouble(values.get(i).toString()));
+            break;
+
+          case "INTEGER":
+            writer.value(Integer.parseInt(values.get(i).toString()));
+            break;
+
+          case "SHORT":
+            writer.value(Short.parseShort(values.get(i).toString()));
+            break;
+
+          case "BYTE":
+            writer.value(Byte.parseByte(values.get(i).toString()));
+            break;
+
+          case "FLOAT":
+            writer.value(Float.parseFloat(values.get(i).toString()));
+            break;
+
+          case "TEXT":
+          case "KEYWORD":
+          case "STRING":
+            writer.value((String) values.get(i));
+            break;
+
+          default:
+            throw new RuntimeException("Invalid data type");
+        }
+      }
+      writer.endObject();
+      writer.close();
+      return stringWriter.toString();
+    } else if (headers != null) {
+      writer.beginObject();
+      for (int i = 0; i < headers.size(); i++) {
+        writer.name(headers.get(i));
+        writer.value(values.get(i));
+      }
+      writer.endObject();
+      writer.close();
+      return stringWriter.toString();
+    } else {
+      throw new RuntimeException("No headers or schema specified");
+    }
```

Comment:
Add blank lines where appropriate for readability

Reply:
Blank lines have been added.